architecture/topics/aws_getting_started_before.adoc

```diff
@@ -0,0 +1,214 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-before-{context}']
+= Before you begin deploying the AWS Service Broker
+
+Before you can get started deploying the AWS Service Broker, make sure you meet the prerequisites described in this topic.
+
+.Prerequisites 
+
+The following items are required to deploy the AWS Service Broker into {product-title}.
+
+{product-title}::
+You must have an {product-title} cluster configured to run the Service Catalog.
+
+////
+AWS Service Broker:: 
+You must have an link:https://aws.amazon.com/partners/servicebroker/[AWS Service Broker] configured with an appropriate registry, such as link:https://hub.docker.com/u/awsservicebroker/[docker.io/awsservicebroker].
+////
+
+AWS Service Instance Policy::
+You must create specific credential and permissions policies in AWS for the AWS Service Broker user or role.
+
+You can create the two required policies using the xref:aws-getting-started-before-policies[provided JSON samples], then assign the policies to the AWS Service Broker user or role or to a separate role, see [Managing Resources Via Assumed Role](/docs/README.md#managing-resources-via-assumed-role).
+
+One policy assigns credentials for provisioning service instances and for broker operations, such as  
+fetching the catalog and reading and writing metadata to DynamoDB.
+
+The second policy assigns permissions for provisioning, binding, and deprovisioning service instances. 
+
+Both of these policies take your AWS account ID, user ID and ARN. You can find this information in AWS. If you have the 
+AWS CLI, use the following command to get your AWS account information:
+
+[source,bash]
+----
+aws sts get-caller-identity
+{
+    "Account": "474574", 
+    "UserId": "AIDAA7ZGME4S", 
+    "Arn": "arn:aws:iam::474574:user/Administrator"
+----
+
+AWS User::
+The AWS Service Broker requires an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html[AWS Identity Access Management (IAM) user] and the associated link:http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[IAM keys] to provision AWS services. The user needs *Programmatic access* and *AWS Management Console* access types and add the two policies you created.
```

Comment:
"The user needs *Programmatic access* and *AWS Management Console* access types and add the two policies you created." You either need to remove "add" or add more words.